modules/RockRuby.cmake

```diff
@@ -21,6 +21,10 @@
 # endif()
 #
 # 
+# If installed to a target directory set the following to true to
+# make it project specific
+# set(RUBY_USE_PROJECT_INSTALL_PREFIX TRUE)
```

Comment:
I am confused now.

Library extensions are installed like this in Rock in practice:
./lib/x86_64-linux-gnu/ruby/2.0.0/frame_helper_ruby.so
./lib/x86_64-linux-gnu/ruby/2.0.0/typelib_ruby.so
./lib/x86_64-linux-gnu/ruby/2.0.0/base_types_ruby.so
./lib/x86_64-linux-gnu/ruby/2.0.0/jpeg_conversion_ext.so
./lib/x86_64-linux-gnu/ruby/2.0.0/servicediscovery_ruby.so

That means there will be a: "require 'frame_helper_ruby'" or similar
Is that what you would say is good 'Ruby' practice or not?

Right now vizkit would install into:
./lib/x86_64-linux-gnu/ruby/2.0.0/vizkit/vizkittypelib.so